### PR TITLE
#63 issue fix

### PR DESCRIPTION
**Pull request by Hyper-ActiveX**
## Issue ticket number and link

Issue Fixes : #63 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: client/yarn.lock
Comment (kanha638) [inline client/yarn.lock:1]: this file is unecessary please remove this

**Review by kanha638 — CHANGES_REQUESTED** [review]
Please add this to my products and subscribed product page also

**Review by kanha638 — CHANGES_REQUESTED** [review]
Please add this to my products and subscribed product page also

**Review by kanha638 — APPROVED** [review]
👍🏻 ready to merge.

**Review by sumit1804 — APPROVED** [review]
changes are completely ok . approved

**Pull request merged**